zhmccli/_cmd_partition.py

```diff
@@ -253,186 +254,146 @@ def partition_dump(cmd_ctx, cpc, partition, **options):
+                 required=False, default=DEFAULT_PROCESSING_WEIGHT,
+                 help='Defines the initial processing weight of CP processors. '
+                 'Default: {d}'.format(d=DEFAULT_PROCESSING_WEIGHT))
+@optgroup.option('--initial-ifl-processing-weight',
+                 type=click.IntRange(MIN_PROCESSING_WEIGHT,
+                                     MAX_PROCESSING_WEIGHT),
+                 required=False, default=DEFAULT_PROCESSING_WEIGHT,
+                 help='Defines the initial processing weight of IFL '
+                 'processors. Default: {d}'.
+                 format(d=DEFAULT_PROCESSING_WEIGHT))
+@optgroup.option('--minimum-ifl-processing-weight',
+                 type=click.IntRange(MIN_PROCESSING_WEIGHT,
+                                     MAX_PROCESSING_WEIGHT),
+                 required=False, default=MIN_PROCESSING_WEIGHT,
+                 help='Represents the minimum amount of IFL processor '
+                 'resources allocated to the partition. '
+                 'Default: {d}'.format(d=MIN_PROCESSING_WEIGHT))
+@optgroup.option('--minimum-cp-processing-weight',
+                 type=click.IntRange(MIN_PROCESSING_WEIGHT,
+                                     MAX_PROCESSING_WEIGHT),
+                 required=False, default=MIN_PROCESSING_WEIGHT,
+                 help='Represents the minimum amount of general purpose '
+                 'processor resources allocated to the partition. '
+                 'Default: {d}'.format(d=MIN_PROCESSING_WEIGHT))
+@optgroup.option('--maximum-ifl-processing-weight',
+                 type=click.IntRange(MIN_PROCESSING_WEIGHT,
+                                     MAX_PROCESSING_WEIGHT),
+                 required=False, default=MAX_PROCESSING_WEIGHT,
+                 help='Represents the maximum amount of IFL processor '
+                 'resources allocated to the partition. '
+                 'Default: {d}'.format(d=MAX_PROCESSING_WEIGHT))
+@optgroup.option('--maximum-cp-processing-weight',
+                 type=click.IntRange(MIN_PROCESSING_WEIGHT,
+                                     MAX_PROCESSING_WEIGHT),
+                 required=False, default=MAX_PROCESSING_WEIGHT,
+                 help='Represents the maximum amount of general purpose '
+                 'processor resources allocated to the partition. '
+                 'Default: {d}'.format(d=MAX_PROCESSING_WEIGHT))
+@optgroup.group('Memory configuration')
+@optgroup.option('--initial-memory', type=int, required=False,
+                 default=DEFAULT_INITIAL_MEMORY_MB,
+                 help='The initial amount of memory (in MiB) when the '
+                 'partition is started. '
+                 'Default: {d} MiB'.format(d=DEFAULT_INITIAL_MEMORY_MB))
+@optgroup.option('--maximum-memory', type=int, required=False,
+                 default=DEFAULT_MAXIMUM_MEMORY_MB,
+                 help='The maximum amount of memory (in MiB) while the '
```

Comment:
```suggestion
                 help='The maximum amount of memory (in MiB) to which the partition's memory allocation can be increased while the '
```

Reply:
I updated the help text as suggested, for partition create and update.